astro/src/components/docs/reference/ConfigurationOptions.astro

```diff
@@ -357,15 +357,19 @@ Display configuration values in format:
         <li><code>production</code></li></ul>
 
     <p>
-      When in the <code>development</code> runtime mode, maintenance mode will interactively assist you configuring the database and connecting to Elasticsearch if configured. Once you move FusionAuth into production, it is recommended that you modify the runtime mode to <code>production</code>. When in <code>production</code> runtime mode maintenance mode will no longer be available to you which means you can be certain that your end users will not find themselves on the database upgrade panel during an Upgrade. When in <code>production</code> mode you will either need to leverage [Silent Mode](/docs/get-started/download-and-install/silent-mode) to automatically apply database migrations or you will need to apply the database migrations yourself (either by hand or via a script of some sort).
+      When in the <code>development</code> runtime mode, maintenance mode will interactively assist you configuring the database and connecting to Elasticsearch if configured. Once you move FusionAuth into production, it is recommended that you modify the runtime mode to <code>production</code>. When in <code>production</code> runtime mode maintenance mode will no longer be available to you which means you can be certain that your end users will not find themselves on the database upgrade panel during an Upgrade. When in <code>production</code> mode you will either need to leverage <a href="/docs/get-started/download-and-install/silent-mode">Silent Mode</a> to automatically apply database migrations or you will need to apply the database migrations yourself (either by hand or via a script of some sort).
     </p>
 
     <span class="text-red-700">Deprecated names:</span>
     <ul><li><code>fusionauth.runtime-mode</code></li></ul>
   </APIField>
 
+  <APIField name="fusionauth-app.search.default-refresh-interval" type="String" defaults="1s" since="1.49.3">
+    The default search index <code>refresh_interval</code>. Bulk operations (bulk user import and reindex) may temporarily relax this value for better performance and then set it back to this value. This value specifies the delay before updated User and Entity data are available to search requests. Time interval strings are acceptable values, although setting this variable is not recommended.
```

Comment:
> Time interval strings are acceptable values

Not sure what this means. Are we trying to specify what format the value needs to be in?

Reply:
I added some examples to help clarify.  I did not want to be exhaustive in the allowable types here b/c changing this value to any interval greater than 1m is a bad idea.